modules/json_content/json_content.module

```diff
@@ -1,10 +1,57 @@
 <?php
 
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\json_content\EntityJsonTask;
+
 /**
  * Implements hook_toolbar().
  */
 function json_content_toolbar_alter(array &$items) {
   // The front page redirects to admin/content and content is generally not
   // viewed outside of the API so this link makes no sense.
   unset($items['home']);
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_entity_operation_alter().
+ */
+function json_content_entity_operation_alter(array &$operations, EntityInterface $entity) {
+  if (isset($operations['edit'])) {
+    $operations['edit']['attributes']['title'] = t('Edit "@title"', [
+      '@title' => $entity->label(),
+    ]);
+  }
+}
+
+/**
+ * Implements hook_local_tasks_alter().
+ */
+function json_content_local_tasks_alter(array &$local_tasks) {
+  $entity_type_manager = \Drupal::entityTypeManager();
+
+  foreach ($local_tasks as $id => $definition) {
+    $matches = [];
+
+    if (preg_match('/^entity\.([a-z0-9_]+)\.canonical$/', $id, $matches)) {
+      $entity_type = $entity_type_manager->getDefinition($matches[1]);
+
+      $bundle_entity_type = $entity_type->getBundleEntityType();
+      if ($bundle_entity_type) {
+        $bundles = $entity_type_manager
+          ->getStorage($bundle_entity_type)
+          ->getQuery()
+          ->execute();
+
+        foreach ($bundles as $bundle) {
```

Comment:
This actually seems to be adding a View JSON link for each bundle - only one of which is valid.
<img width="801" alt="edit_image_da___drush_site-install" src="https://user-images.githubusercontent.com/1536654/34386561-440984d0-eaf7-11e7-8645-9fdee18e3a4b.png">